chef-provisioning.gemspec

```diff
@@ -19,6 +19,7 @@ Gem::Specification.new do |s|
   s.add_dependency 'inifile', '~> 2.0'
   s.add_dependency 'cheffish', '~> 1.1'
   s.add_dependency 'winrm', '~> 1.3'
+  s.add_dependency "mixlib-install",  ">= 0.4", "< 1.0"
```

Comment:
You could use `~> 0.4` and it will accomplish the same thing as what you have here